### PR TITLE
Add documentation for the InfluxDB import script

### PR DESCRIPTION
**Pull request by janLo**
**Description:**

This adds documentation for a script to import recorded events from a recorder database
to a InfluxDB instance. This can be useful for initial importing after
setup of a InfluxDB.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#7254

### CODE REVIEW

File: source/_components/influxdb.markdown [hunk 1]
```diff
@@ -98,6 +98,62 @@ optional arguments:
...
+  -d dbname, --dbname dbname
+                        InfluxDB database name
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -c path_to_config_dir, --config path_to_config_dir
+                        Directory that contains the Home Assistant
+                        configuration
+  --uri URI             Connect to URI and import (if other than default
+                        swlite)eg: mysql://localhost/homeassistant
```
Comment (MartinHjelmare) [inline source/_components/influxdb.markdown]: "default sqlite"

File: source/_components/influxdb.markdown [hunk 2]
```diff
@@ -98,6 +98,62 @@ optional arguments:
...
+  -D default_measurement, --default-measurement default_measurement
+                        Store all your points in the same measurement
+  -o override_measurement, --override-measurement override_measurement
+                        Store all your points in the same measurement
+  -e exclude_entities, --exclude_entities exclude_entities
+                        Comma separated list of excluded entities
+  -E exclude_domains, --exclude_domains exclude_domains
+                        Comma separated list of excluded domains
+  -S, --simulate        Do not write points but simulate preprocessing
+                        and print print statistics
```
Comment (janLo) [inline source/_components/influxdb.markdown]: Fixed. :)

File: source/_components/influxdb.markdown (hunk 1 above)
Comment (janLo) [inline source/_components/influxdb.markdown]: Fixed. :)

**Comment by MartinHjelmare** [comment]
Merge when parent PR is merged.

**Comment by fabaff** [comment]
I guess that now it's time for a separate script section in the docs. https://home-assistant.io/docs/tools/scripts/ seems no longer to meet the requirements if the descriptions are getting more comprehensive.

**Pull request merged**